scripts/unit.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+: "${WHAT:?}"
+
+ROOT_DIR=$(dirname "${BASH_SOURCE[0]}")/..
+TARGET_NAME=${TARGET_NAME:="unit"}
+
+pushd "${ROOT_DIR}/staging/${WHAT}"
+make ${TARGET_NAME}
+popd
```

Comment:
This is a bit awkward but this should work when calling that `unit` driver make target in the CI prow configurations.